### PR TITLE
Fix documentation on docker sandboxes

### PR DESCRIPTION
**Pull request by junr03**

### CODE REVIEW

File: examples/front-proxy/README.md [hunk 1]
```diff
@@ -1,38 +1,2 @@
...
-
-```shell
-$ pwd
-src/envoy/
-$ docker build -f example/Dockerfile-envoy-image -t envoy .
-```
-
-Now you can use this `envoy` image to build the demo if you change the `FROM`
-line in `Dockerfile-frontenvoy` and `Dockerfile-service`
+to the [envoy docs](https://envoydocs-staging.lyft.net/docs/install/sandboxes.html#front-proxy)
```
Comment (mattklein123) [inline examples/front-proxy/README.md]: public link

File: examples/grpc-bridge/README.md [hunk 2]
```diff
@@ -1,31 +1,2 @@
...
-# set a key
-docker-compose exec python /client/client.py set foo bar
-=> setf foo to bar
-
-# get a key
-docker-compose exec python /client/client.py get foo
-=> bar
-```
+To learn about this sandbox and for instructions on how to run it please head over
+to the [envoy docs](https://envoydocs-staging.lyft.net/docs/install/sandboxes.html#grpc-bridge)
```
Comment (mattklein123) [inline examples/grpc-bridge/README.md]: public link

File: docs/install/sandboxes.rst [hunk 3]
```diff
@@ -3,7 +3,47 @@
...
+
+Locally building a docker image with an envoy binary
+----------------------------------------------------
+
+The following steps guide you through building your own envoy binary, and
+putting that in a clean ubuntu container.
+
+**Step 1: Build Envoy**
+
+Using ``lyft/envoy-buld`` you will compile envoy.
```
Comment (mattklein123) [inline docs/install/sandboxes.rst]: typo

File: docs/install/sandboxes.rst [hunk 4]
```diff
@@ -3,7 +3,47 @@
 Sandboxes
 =========
 
-To get a flavor of what Envoy has to offer, we are releasing a
+The docker-compose sandboxes give you different environments to test out Envoy's
+features. As we gauge people's interests we will add more sandboxes demonstrating
+different features.
+
+Locally building a docker image with an envoy binary
```
Comment (mattklein123) [inline docs/install/sandboxes.rst]: This section should come last, as it should not be required to run the demos. The demo docs should start with a quick start on how to get up and running quickly with already pushed images.

File: docs/install/sandboxes.rst [hunk 5]
```diff
@@ -226,3 +266,47 @@ statistics. For example inside ``frontenvoy`` we can get::
...
 fulfilled by them, information about http ingress, and a plethora of other useful
 stats.
+
+gRPC bridge
+-----------
+
+Envoy gRPC
+~~~~~~~~~~
+
+The gRPC bridge sandbox is an example usage of Envoy's :ref:`gRPC bridge filter <config_http_filters_grpc_bridge>`.
```
Comment (mattklein123) [inline docs/install/sandboxes.rst]: 100 col limit

File: docs/install/sandboxes.rst [hunk 6]
```diff
@@ -226,3 +266,47 @@ statistics. For example inside ``frontenvoy`` we can get::
...
+Included in the sandbox is a gRPC in memory Key/Value store with a Python HTTP
+client. The Python client makes HTTP/1 requests through the Envoy sidecar
+process which are upgraded into HTTP/2 gRPC requests. Response trailers are then
+buffered and sent back to the client as a HTTP/1 header payload.
+
+Another Envoy feature demonstrated in this example is Envoy's ability to do authority
+base routing via its route configuration.
+
+Building the Go service
```
Comment (mattklein123) [inline docs/install/sandboxes.rst]: this should go last, images should already be pushed so people can quickly get started.

File: examples/front-proxy/README.md (hunk 1 above)
Comment (junr03) [inline examples/front-proxy/README.md]: fixed

File: examples/grpc-bridge/README.md (hunk 2 above)
Comment (junr03) [inline examples/grpc-bridge/README.md]: fixed

File: docs/install/sandboxes.rst (hunk 4 above)
Comment (junr03) [inline docs/install/sandboxes.rst]: moved

File: docs/install/sandboxes.rst (hunk 3 above)
Comment (junr03) [inline docs/install/sandboxes.rst]: fixed

File: docs/install/sandboxes.rst (hunk 5 above)
Comment (junr03) [inline docs/install/sandboxes.rst]: fixed

File: docs/install/sandboxes.rst (hunk 6 above)
Comment (junr03) [inline docs/install/sandboxes.rst]: moved

**Pull request merged**